Project Access update comments (#5)
ScilifelabDataCentre/dds_workshop: docs/2_registration_project_access_session.md

![Screenshot of a sucessfull project creation](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-project-create.svg)

> > You can always retrieve the the **Project ID** using the command `dds ls` and then setting meaningful `--title` and `--description` will prove useful for you.

Reviewer: Can we rephrase this? For example.
```suggestion
> You can always retrieve the the **Project ID** using the command `dds ls`. Setting a meaningful `--title` and `--description` when creating the project will be useful when searching your list of ongoing projects.
```

> There are other flags that can be passed down, like `--owner` to add a user as a Project Owner (researcher with elevated privileges), or `--researcher` to automatically invite researcher users to the project. The list and description can be found on the documentation link just above.

> # Upload data

Reviewer 2: I think this content is good but veers very much into the 2nd session of 'project life cycle'. Could we move from here onwards into the (currently empty) lifecycle session?

Reviewer: Agreed.

Author: Wouldn't then this file be too empty?

dds data put --project "<Project ID>" --source "<File or directory to upload>"
~~~

> > There are a number of other optional flags, which can be found on the [documentation](https://scilifelabdatacentre.github.io/dds_cli/data/#dds-data-put) link here.

Reviewer 2: ```suggestion
> There are a number of other optional flags, which can be found in the [documentation](https://scilifelabdatacentre.github.io/dds_cli/data/#dds-data-put) link here.
```

> data/example_directory_2/sub_directory_2: example_file_3.txt     
> example_file_4.txt

> On windows, you can use the `dir` command or manually explore the structure through the file explorer.

Reviewer 2: ```suggestion
On Windows, you can use the `dir` command or manually explore the structure through the File Explorer.
```

 - [ ] TASK: Upload the files in the data folder. Remember to change the **project ID** and the **source** from the command above

> When an upload is interrupted, you can resume it later on. The system will detect which files are already on the cloud and only ulpload the remaining ones. In order to replace them you must specify the `--overwrite` flag.

> If you have forgotten the project ID, you can list all active projects you have access with the ls command, [documentation](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-ls) here.

Reviewer 2: ```suggestion
If you have forgotten the project ID, you can list all active projects to which you have access with the `ls` command, [see documentation](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-ls) here.
```

~~~
dds project ls
~~~

> NOTE: As of today, DDS can have performance issues and upload failures when uploading many small files. We are working
> on fixing it, but in the meantime, we recommend that you archive (with ZIP or RAR) deliveries when the delivered data consists of hundreds or thousands of small files.

> > On Mac/Linux this can be archieved with 

Reviewer 2: ```suggestion
> On Mac/Linux this can be achieved with:
```

![Message shown after project release](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-project-status-release.svg)
##

> After releasing the project, any curent researcher associated with the project will receieve a notification by email.

Reviewer 2: ```suggestion
After releasing the project, any current researcher associated with the project will receive a notification by email.
```

Where spf.txt is the path to a file you have created with the following format:

~~~
data/example_directory_1/sub_directory_1/
data/example_directory_2/sub_directory_2/example_file_3.txt
data/example_directory_2/example_file_5.txt
~~~

 - [ ] TASK: Release the project after making sure all the data was uploaded, remember to specify the correct project ID
 - [ ] TASK: Download the project contents, and verify that they are the correct. You can download data with your unit admin account.

> #### Researchers

Reviewer 2: I think this part should stay in this session but all of the upload/download data stuff should move the project life cycle session.

Author: I prefer having one complete file with all the instructions rather than several small ones. 
I am having trouble understanding our goal on this because I think everyone has an idea. 

Currently, we only have one slide related to the demo, and it is before explaining project life cycle. Are we doing several demos, one of each file? One big demo at the end? I am quite confused at this point

Reviewer 3: Let's get the small change suggestions done without moving things around in this PR.

Reviewer 2: Yeah, maybe if we get the small changes done I can make a new PR afterwards as I think it's easier to just show the proposal.

Author: So. Do I merged this two requests then?

Reviewer 2: If everything except the more major 'moving some content to a different file' is done, then I would say yes.
